mindjunkies/home/tests/test_enrollment_tags.py

```
@@ -0,0 +1,75 @@
+        result = get_enrollment(self.course, None)
+        self.assertIsNone(result)
+
+    def test_template_usage(self):
+        """Test the filter in a template context"""
+        context = template.Context({
+            "course": self.course,
+            "user": self.user,
+        })
+        template_str = "{% load enrollment_tags %}{{ course|get_enrollment:user }}"
+        t = template.Template(template_str)
+         # Check Enrollment object representation
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)